app/serializers/api/participant_serializer.rb

```diff
@@ -0,0 +1,50 @@
+module API
+  class ParticipantSerializer < Blueprinter::Base
+    class AttributesSerializer < Blueprinter::Base
+      exclude :id
+
+      field(:full_name) { |p, _| [p.trainee.teacher.trs_first_name, p.trainee.teacher.trs_last_name].join(' ') }
+      field(:trn) { |p, _| p.trainee.teacher.trn }
+
+      field(:ecf_enrolments) do |object, options|
+        object.trainee.training_periods.map do |training_period|
+          next unless training_period.lead_provider == options[:lead_provider]
+
+          {
+            training_record_id: training_period.id,
```

Comment:
we need to think about if training record will match participant profile id in ECF1, or will it be different? where would we put participant profile id going forward as training record is closer to an induction record than a profile?